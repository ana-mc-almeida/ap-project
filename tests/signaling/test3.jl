include("common.jl")

handling(LineEndLimit => (c) -> println()) do
    print_line("Hi, everybody! How are you feeling today?")
end