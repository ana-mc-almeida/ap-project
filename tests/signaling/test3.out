Hi, everybody! How a
re you feeling today
?